src/sidebar/components/annotation.js

```diff
@@ -569,8 +569,6 @@ function AnnotationController(
     }
     return self.group().type !== 'private';
   };
-
-  init();
```

Comment:
The `init` function has been converted to an `$onInit` method on the controller which Angular invokes automatically, so we don't need to call it ourselves.